invoice/src/main/java/org/killbill/billing/invoice/api/InvoiceApiHelper.java

```diff
@@ -129,10 +137,10 @@ public List<InvoiceItem> dispatchToInvoicePluginsAndInsertItems(final UUID accou
             if (success) {
                 for (final Invoice invoiceForPlugin : invoicesForPlugins) {
                     final DefaultInvoice refreshedInvoice = new DefaultInvoice(dao.getById(invoiceForPlugin.getId(), internalCallContext));
-                    invoicePluginDispatcher.onSuccessCall(targetDate, refreshedInvoice, existingInvoices, isDryRun, isRescheduled, context, priorCallResult.getPluginProperties(), internalCallContext);
```

Comment:
Don't we need to keep track of the adjusted properties and pass them to the next plugin?

Reply:
Not sure if this comment is still valid: For reference, we are keeping track of the adjusted properties in `pluginProperties` and this is what we pass for each plugin invocation.